src/register_target.cpp

```diff
@@ -1,5 +1,6 @@
-#include <migraphx/register_target.hpp>
+#include <algorithm>
```

Comment:
I dont think this additional header is needed.

Reply:
Removed.